src/lib/connectors/notify.js

```diff
@@ -0,0 +1,37 @@
+const NotifyClient = require('notifications-node-client').NotifyClient;
+
+const messageTypes = {
+  email: 'email'
+};
+
+/**
+ * Gets the Notify API key.
+ * @param  {Object} env  Node environment
+ * @param  {String} type Message type email|letter
+ * @return {String}      Notify API key
+ */
+const getKey = (env, type) => {
+  // Always use live key in production environment
+  if (env.NODE_ENV === 'production') {
+    return env.LIVE_NOTIFY_KEY;
+  }
+  // In other environments, use whitelist key for email/SMS, and test key for anything else
+  return [messageTypes.email, messageTypes.sms].includes(type)
```

Comment:
It looks like `messageTypes.sms` might be undefined here

Reply:
SMS message type removed